tests/interface/TWPrivateKeyTests.cpp

```diff
@@ -85,24 +85,6 @@ TEST(TWPrivateKeyTests, PublicKey) {
     }
 }
 
-TEST(TWPrivateKeyTests, ClearMemory) {
-    auto privKey = "afeefca74d9a325cf1d6b6911d61a65c32afa8e02bd5e78e2e4ac2910bab45f5";
-    auto privKeyData = TW::parse_hex(privKey);
-    auto data = WRAPD(TWDataCreateWithBytes(privKeyData.data(), privKeyData.size()));
-    auto privateKey = TWPrivateKeyCreateWithData(data.get());
-    auto ptr = privateKey->impl.bytes.data();
-    ASSERT_EQ(privKey, TW::hex(TW::data(ptr, 32)));
-
-    TWPrivateKeyDelete(privateKey);
```

Comment:
can we keep this test? inside `TWPrivateKeyDelete` is still `delete pk`, make sure it's cleared from C interface call

Reply:
This test was tricky, I struggled with it already.  It tests the content of the memory after being freed, which is not valid.  The test result was sporadic, as that memory is often reused quickly (hence the probabilistic check, not compare to all 0s).  The sanitizer caught the invalid memory access done by the test.
'Zero memory before deallocation' is a security feature, but impossible to test properly.